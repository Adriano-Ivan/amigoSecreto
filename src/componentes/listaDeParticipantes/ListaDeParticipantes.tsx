import { useListaDeParticipantes } from "../../state/hooks/useListaDeParticipantes";
import styles from "./ListaDeParticipantes.module.css";

const ListaDeParticipantes = () => {
    const participantes = useListaDeParticipantes();

    return (
        <ul className={`${styles['lista-de-participantes']}`}>
            {
                participantes.map((participante) =>{
                    return <li key={participante}>{participante}</li>
                })
            }
        </ul>
       
    );
}

export default ListaDeParticipantes;