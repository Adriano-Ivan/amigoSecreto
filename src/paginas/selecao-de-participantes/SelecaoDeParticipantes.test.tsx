import { render } from "@testing-library/react";
import {RecoilRoot} from "recoil";
import  React from "react";
import ListaDeParticipantes from "../../componentes/listaDeParticipantes/ListaDeParticipantes";

const mockNavegacao = jest.fn();

jest.mock("react-router-dom", () => {
    return {
        useNavigate: () => mockNavegacao
    }
});

describe("página de configuração", () => {

    test("deve ser renderizada corretamente", () => {
        const {container }  = render(
            <RecoilRoot >
                <ListaDeParticipantes/>
            </RecoilRoot>
        );

        expect(container).toMatchSnapshot();
    });
});