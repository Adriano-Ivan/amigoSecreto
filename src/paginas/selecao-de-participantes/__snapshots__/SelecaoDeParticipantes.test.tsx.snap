// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`página de configuração deve ser renderizada corretamente 1`] = `
<div>
  <ul
    class="lista-de-participantes"
  />
</div>
`;
